playground/nextjs/src/pages/index.tsx

```diff
@@ -1,21 +1,25 @@
 import { Mounter } from '@/components/Mounter';
+import type { NovuUI, NovuUIOptions } from '@novu/js/ui';
 
 export default function Home() {
   return (
     <div className="h-screen w-full bg-neutral-100">
       <Mounter
         mount={(el) => {
-          const NovuUI = require('@novu/js/ui').NovuUI;
-          const ui = new NovuUI({
+          //require here as it won't work in SSR.
+          const { NovuUI } = require('@novu/js/ui');
```

Comment:
Duplicate import, NovuUI is imported at the top

Reply:
I am just importing the type so I can cast it! We can't import in SSR so it will error if we do it on top. This will be handled by the react/nextjs package once we make those.